src/components/confirm/confirm-test.stories.mdx

```diff
@@ -88,6 +88,14 @@ export const ConfirmStory = ({
 
 # Confirm
 
+  <Link
+    href="https://zeroheight.com/2ccf2b601/p/497e21-dialog"
+    target="_blank"
+    rel="noreferrer noopener"
+  >
+    Product Design System component
+  </Link>
+
```

Comment:
```suggestion
```
I would remove this link after since it does not direct the consumer to the DS docs for a component with a similar name 😄